letsdrill.rb

```diff
@@ -1,24 +1,41 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+
+  if integer >= 90
+    "A"
+  elsif integer >= 80
+    "B"
+  elsif integer >= 70
+    "C"
+  elsif integer >= 60
+    "D"
+  else
+    "F"
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  shortest = array[0]
+  array.each do |s|
+    if s.length < shortest.length
+      shortest = s
+    end
+  end
+
+  return shortest
```

Comment:
Take a look at the documentation for ruby array methods, and see if you can't pull this off with a single, simple line.  Also, on the current approach, idiomatic ruby is to omit `return` unless it's necessary.